src/inmanta/module.py

```diff
@@ -138,10 +138,10 @@ def parse(cls: Type[TInmantaModuleRequirement], spec: str) -> TInmantaModuleRequ
         return cls(Requirement.parse(spec))
 
 
-@stable_api
-class InvalidModuleException(CompilerException):
+class CompilerExceptionWithExtendedTrace(CompilerException):
```

Comment:
This is somewhat similar to `ExternalException`. 

With the difference that `ExternalException` is a `RuntimeException` and this one is not.
(just making the remark, this is not actionable)

Reply:
I didn't use the `ExternalException`, because it's  `RuntimeException`. The docstring of the `RuntimeException` mentions _"Baseclass for exceptions raised by the compiler after parsing is complete."_. This condition is not satisfied for this specific use case.